docs/zkapps/lightnet.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: Lightnet zkApp Testing
+hide_title: true
+description: Use lightweight Mina blockchain network (Lightnet) for testing on a local blockchain before you test with a live network. 
+keywords:
+  - zkApp CLI
+  - lightnet
+  - zkApp testing
+  - local testing
+  - using zk lightnet
+---
+
+# Lightnet zkApp Testing
+
+The most efficient way to deploy and run a lightweight Mina blockchain network for testing purposes is to spin up a lightweight Mina network (Lightnet) in a single Docker container. Lightnet is a resource-efficient solution with fast startup and syncing times.
+
+Lightnet lets you test your zkApp locally on an accurate representation of Mina blockchain before you test with a live network.
+
+Using Lightnet provides the following benefits:
+
+- Reduces the time from ideation to launch by letting you test your applications and changes against a close-to-real Mina network.
+- Starts a single-node network or a multi-node network with diverse types of participants.
+- Creates and funds accounts so that you can deploy and interact with your zkApp. The genesis ledger is configured with 1000 pre-funded accounts with a 1550 MINA balance on each account.
+- Optionally runs the Mina [archive node](/node-operators/archive-node).
+- Provides the Mina accounts manager helper tool so you can automate account retrieval.
+- Launches the lightweight Mina Explorer in a web browser so you can monitor transactions.
+
+:::info
+
+Use of Lightnet blockchain is appropriate for local development and testing only. It is not intended to replicate all aspects of the full network.
+
+:::
+
+## Prerequisites
+
+Docker Engine is required for your environment. 
+
+1. [Install Docker Engine](
+https://docs.docker.com/engine/install/).
+1. Start Docker Engine. 
+
+Docker Engine must be running before you can start the Docker container.
+
+## Start a single node network
+
+To start a single node network with default behavior:
+
+```sh
+zk lightnet start
+```
+
+This command uses default settings that successfully serve the majority of testing requirements:
```

Comment:
```suggestion
This command performs the following operations using default settings that successfully serve the majority of testing requirements:
```
Could probably be better still. The original version was implying that what follows is a list of testing requirements but it's actually a list of things the command does. Trying to reflect that more clearly